ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md

```diff
@@ -3961,6 +3961,51 @@ interface Ethernet81/2
 interface Ethernet81/10
    description isis_port_channel_member
    channel-group 110 mode active
+!
+interface Ethernet82
+   description Switchport_tap_tool
+   switchport tap native vlan 10
+   switchport tap identity 3 inner 5
+   switchport tap mac-address dest 01:00:00:00:00:00 src 01:23:45:67:89:ab
+   switchport tap mpls pop all
+   switchport tap encapsulation gre protocol 0x0001 strip
+   switchport tap encapsulation gre protocol 0x0002 feature header length 3 strip
+   switchport tap encapsulation gre protocol 0x0003 feature header length 2 strip re-encapsulation ethernet
+   switchport tap encapsulation gre protocol 0x0004 strip re-encapsulation ethernet
+   switchport tap encapsulation gre strip
+   switchport tap encapsulation gre destination 1.1.1.1 source 1.1.1.2 protocol 0x0000 strip
+   switchport tap encapsulation gre destination 1.1.1.1 source 1.1.1.2 strip
+   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x0010 strip
+   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x0011 feature header length 2 strip re-encapsulation ethernet
+   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x0012 strip re-encapsulation ethernet
+   switchport tap encapsulation gre destination 2.1.1.3 source 2.1.1.4 strip
+   switchport tool mpls pop all
```

Comment:
you will need to use multiple interfaces

```
site3-wan1(config-if-Et2)#   switchport tool mpls pop all
site3-wan1(config-if-Et2)#   switchport tool encapsulation vn-tag strip
! 'mpls pop' has already been configured and takes precedence over 'vn-tag strip'.
site3-wan1(config-if-Et2)#   switchport tool encapsulation dot1br strip
! 'mpls pop' has already been configured and takes precedence over 'dot1br strip'.

Reply:
yes it gives this warning, which is mentioned in the schema description, but the config is applied in running-config with both commands -

```
s1-leaf2(config-if-Et10)#show active
interface Ethernet10
   description Switchport_tap_tool
   switchport tap native vlan 10
   switchport tap identity 3 inner 5
   switchport tap mac-address dest 01:00:00:00:00:00 src 01:23:45:67:89:ab
   switchport tap encapsulation gre protocol 0x1 strip
   switchport tap encapsulation gre protocol 0x2 feature header length 3 strip
   switchport tap encapsulation gre protocol 0x3 feature header length 2 strip re-encapsulation ethernet
   switchport tap encapsulation gre protocol 0x4 strip re-encapsulation ethernet
   switchport tap encapsulation gre destination 1.1.1.1 source 1.1.1.2 protocol 0x0 strip
   switchport tap encapsulation gre destination 1.1.1.1 source 1.1.1.2 strip
   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x10 strip
   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x11 feature header length 2 strip re-encapsulation ethernet
   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x12 strip re-encapsulation ethernet
   switchport tap encapsulation gre destination 2.1.1.3 source 2.1.1.4 strip
   switchport tap mpls pop all
   switchport tool mpls pop all
   switchport tool encapsulation vn-tag strip
   switchport tool encapsulation dot1br strip
   switchport tap allowed vlan 25
   switchport tool allowed vlan 23
```